openmc/mgxs_library.py

```diff
@@ -701,12 +700,12 @@ def add_xsdata(self, xsdata):
         # Check the type
         if not isinstance(xsdata, XSdata):
             msg = 'Unable to add a non-XSdata "{0}" to the ' \
-                  'MGXSLibraryFile'.format(xsdata)
+                  'MGXSLibrary instance'.format(xsdata)
             raise ValueError(msg)
 
         # Make sure energy groups match.
         if xsdata.energy_groups != self._energy_groups:
-            msg = 'Energy groups of XSdata do not match that of MGXSLibraryFile!'
+            msg = 'Energy groups of XSdata do not match that of MGXSLibrary!'
```

Comment:
Just a thought - we don't use exclamation points in any of our other errors, but perhaps this is a particularly special case that warrants one.